base/applications/taskmgr/graphctl.c

```diff
@@ -407,6 +419,52 @@ void GraphCtrl_Paint(TGraphCtrl* this, HWND hWnd, HDC dc)
     DeleteDC(memDC);
 }
 
+void GraphCtrl_ShiftGrid(TGraphCtrl* this)
+{
+    RECT rectCleanUp;
+    HPEN oldPen, solidPen;
+    int i;
+
+    if (this->m_dcGrid == NULL)
+        return;
+
+    solidPen = CreatePen(PS_SOLID, 0, this->m_crGridColor);
+
+    /*  BitBlt it to itself to scroll */
```

Comment:
```suggestion
    /*  Scroll the grid left: BitBlt it to itself */
```

Reply:
Accepted. English isn't my primary language so there would be wording issues. Thanks